lib/ui/screen/login_screen.dart

```diff
@@ -0,0 +1,106 @@
+import 'package:dio/dio.dart';
+import 'package:flutter/material.dart';
+import 'package:fluttertoast/fluttertoast.dart';
+
+import '../../data/remote/dio_helper.dart';
+
+class LoginScreen extends StatelessWidget {
+
+  LoginScreen({super.key});
+
+  final emailController = TextEditingController();
+  final passwordController = TextEditingController();
```

Comment:
bad performance pls dispose the controller in dispose method , or prefer to use form key and onSave function  instead of use controller